.github/workflows/opensearch-notebooks-test-and-build-workflow.yml

```diff
@@ -3,9 +3,8 @@ name: Test and Build OpenSearch Notebooks Backend Plugin
 on: [push, pull_request]
 
 env:
-  PLUGIN_NAME: notebooksDashboards
   OPENSEARCH_VERSION: '1.0'
-  COMMON_UTILS_VERSION: main
+  COMMON_UTILS_VERSION: '1.0'
```

Comment:
why not just use `opensearch_version` for common-utils

Reply:
common utils branch would be cut after opensearch due to its dependency on opensearch, so they might be different. we'll manually separate the two refs so our CI won't fail if that happens